docs/release-notes/RELEASE-NOTES-0.10.0.md

```diff
@@ -2,8 +2,7 @@
 
 [This release][this-release] is focused on an encrypted Peer-to-Peer protocol and consistent serialization of chain objects.
 It:
-* Moves Peer-to-Peer communication from HTTP to encrypted channels (Noise is used to negoitiate cryptographic material).
-* Introduces a consistent and deterministic serialization using RLP, moving away from MessagePack and internal format.
```

Comment:
Did you mean to remove the RLP line?

Reply:
+1

Reply:
Yes, since you wrote the same thing a few lines further down...